src/metabase/search/spec.clj

```diff
@@ -44,7 +44,8 @@
    :dashboard-id        :int
    :dashboardcard-count :int
    :database-id         :pk
-   :id                  :text
+   ;; MySQL cannot have `:text` column in an index
```

Comment:
I'm not sure there was actually a good reason to make this text anyway, in practice the strings will be pretty short.

Do varchar columns reserve the full space for every row? If this is the case for Postgres we'll probably want a smaller size.

Reply:
In case of postgres `varchar(100)` is just a synonym for `text` with a constraint for 100 chars (it's all TOAST anyway). I don't know enough about MySQL but I feel like we can ignore the question. 😁